app/models/katello/repository.rb

```diff
@@ -413,7 +413,7 @@ def full_path(smart_proxy = nil, force_http = false)
       elsif ostree?
         "#{scheme}://#{pulp_uri.host.downcase}/pulp/content/web/#{relative_path}"
       elsif ansible_collection?
-        "#{scheme}://#{pulp_uri.host.downcase}/pulp_ansible/galaxy/#{relative_path}/api"
+        "#{scheme}://#{pulp_uri.host.downcase}/pulp_ansible/galaxy/#{relative_path}/api/"
       else
```

Comment:
Allows users to copy paste this path as remote for other pulp instance, (ex: hub) with a trailing slash.